### PR TITLE
update to julia >= 1.1

### PR DESCRIPTION
**Pull request by gmassman**
* update syntax
* use Project.toml instead of REQUIRE
* remove Requests dependency and functions to access EBI over HTTP
* support for julia >= 1.1 due to Twiddle dependency

### CODE REVIEW

File: src/diff.jl [hunk 1]
```diff
@@ -44,15 +44,15 @@ end
 function open_stream( filename )
    fopen = open( filename, "r" )
    if isgzipped( filename )
-      stream = ZlibInflateInputStream( fopen, reset_on_end=true, bufsize=6_000_000 )
```
Comment (timbitz) [inline src/diff.jl]: Is this parameter deprecated now?

File: src/diff.jl [hunk 2]
```diff
@@ -111,24 +108,24 @@ function process_psi_line( streams::Vector{BufferedStreams.BufferedInputStream};
...
+                                     b::Vector{BufferedStreams.BufferedInputStream};
                                      min_samp=1, min_reads=5, amt=0.0, size=1000 )
    io = open( outfile, "w" )
    stream = ZlibDeflateOutputStream( io )
    output_diff_header( stream )
    while true # go through all lines until we hit eof
       a_event,a_complex,a_entropy,a_post = process_psi_line( a, min_reads=min_reads, size=size )
       b_event,b_complex,b_entropy,b_post = process_psi_line( b, min_reads=min_reads, size=size )
-      if a_event == split( "", "" ) && b_event == split( "", "" )
+      if a_event == split( "", "" ) || b_event == split( "", "" )
```
Comment (timbitz) [inline src/diff.jl]: Good catch! Not sure why I had && here?

Comment (timbitz) [reply src/diff.jl]: Or was that my change? ;)

**Comment by timbitz** [comment]
@gmassman, This looks okay I think -- we should probably just do a quick performance benchmark on real data vs. v0.11, update the docs about 0.6.4, and get codecov to run.

File: src/diff.jl (hunk 1 above)
Comment (gmassman) [reply src/diff.jl]: Not sure why that was deleted? I just checked, seems like it's still available so I'm going to add it back.

File: src/diff.jl (hunk 2 above)
Comment (gmassman) [reply src/diff.jl]: Yeah that was your change I copied from the other branch.

**Pull request closed without merging**